### PR TITLE
Manage Zeroclipboard buttons without Flash

### PR DESCRIPTION
**Pull request by tarebyte**
Currently I use Firefox, but do not have Flash installed.

I always forget and try to click on the `Gemfile` and `Install` copy buttons, and then I realize it won't work for me.

My current solution is to remove them from the textbox when flash is not present. This is what is looks like without them.

screen shot 2015-02-15 at 23 28 08

I still think there still needs some styling work to do, but I wanted to get a feel if this was a plausible solution that people would like.

Thanks!

### CODE REVIEW

**Comment by arthurnn** [comment]
I think we should handle this in a similar way that GitHub handles it(on the copy button for the git url).

**Comment by tarebyte** [comment]
@arthurnn I can make that happen

**Comment by tarebyte** [comment]
@arthurnn I moved the span tag with the text to a readonly input form.

This is what it looks like on Chrome
screen shot 2015-02-16 at 00 34 14

This is what it looks like on Firefox
screen shot 2015-02-16 at 00 34 55

File: app/views/rubygems/show.html.erb [hunk 1]
```diff
@@ -123,7 +123,7 @@
         <h2 class="gem__ruby-version__heading t-list__heading">
           <%= t '.install' %>:
           <div class="gem__code-wrap">
-            <span class="gem__code"><%= @latest_version.to_install %></span>
+            <input type="text" class="gem__code" value="<%= @latest_version.to_bundler %>" readonly="readonly">
```
Comment (arthurnn) [inline app/views/rubygems/show.html.erb]: should here be `@latest_version.to_install` ?

Comment (tarebyte) [inline app/views/rubygems/show.html.erb]: Yup! That's a copy paste error on my part, will fix in a minute.

**Comment by tarebyte** [comment]
@arthurnn Here is how it looks now on Firefox

screen shot 2015-02-16 at 12 24 42

**Comment by arthurnn** [comment]
thanks :heart: 

merged via https://github.com/rubygems/rubygems.org/commit/ab4b654bb8a1467efc96246746f38f4ae682a9b9

**Comment by zenorocha** [comment]
Why don't you guys use [clipboard.js](http://zenorocha.github.io/clipboard.js/) instead? :)

**Pull request closed without merging**